trunk/SUAVE/Analyses/Mission/Segments/Transition/Lift_Cruise_Optimized.py

```diff
@@ -0,0 +1,172 @@
+## @ingroup Analyses-Mission-Segments-Transition
+# Lift_Cruise_Optimized.py
+#
+# Created:  Apr 2018, M. Clarke
+# Modified:
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# SUAVE imports
+from SUAVE.Analyses.Mission.Segments import Aerodynamic
+from SUAVE.Analyses.Mission.Segments import Conditions
+
+from SUAVE.Methods.Missions import Segments as Methods
+
+from SUAVE.Analyses import Process
+
+# Units
+from SUAVE.Core import Units
+import SUAVE
+
+# ----------------------------------------------------------------------
+#  Segment
+# ----------------------------------------------------------------------
+
+## @ingroup Analyses-Mission-Segments-Transition
+class Lift_Cruise_Optimized(Aerodynamic):
+    """ Optimize your transition segment. This is useful if you're not sure how your vehicle should transition.
+        You can set any conditions parameter as the objective, for example setting time to transition or vehicle mass
+        
+        The ending airspeed is an optional parameter.
+        
+        This segment takes far longer to run than a normal segment. Wrapping this into a vehicle optimization
+        has not yet been tested for robustness.
+        
+    
+        Assumptions:
+        Can use SNOPT if you have it installed through PyOpt. But defaults to SLSQP through 
+        Runs a linear true airspeed mission first to initialize conditions.
+        
+        Source:
+        None
+    """          
+    
+    def __defaults__(self):
+        """ This sets the default solver flow. Anything in here can be modified after initializing a segment.
+    
+            Assumptions:
+            None
+    
+            Source:
+            N/A
+    
+            Inputs:
+            None
+    
+            Outputs:
+            None
+    
+            Properties Used:
+            None
+        """          
+        
+        # --------------------------------------------------------------
+        #   User inputs
+        # --------------------------------------------------------------
+        self.altitude                = None
+        self.air_speed_start         = None
+        self.air_speed_end           = None
+        self.acceleration            = None
+        self.pitch_initial           = None
+        self.pitch_final             = None     
+        self.objective               = None # This will be a key
+        self.minimize                = True
+        self.lift_coefficient_limit  = 1.2 
```

Comment:
No, we didn't. The Optimized Climb Segment still resides in the Mission/Segments/Climb folder. I have yet to create a regression test for either of the optimized segments. They currently take too long. I personally think they should kept in the same folder with their respective segments since code/calculations are similar.

Reply:
I removed this script